viahtml/hooks/hooks.py

```diff
@@ -18,6 +19,14 @@
 ]
 
 
+def query_param_as_bool(value: str) -> bool:
+    """Parse a boolean query param value. Any non-zero number is considered true."""
+    try:
+        return bool(int(value))
+    except ValueError:
+        return False
```

Comment:
We can't use `asbool` from Pyramid here because this is not a Pyramid app. There is an `asbool` in `viahtml.app`, which is currently used for reading app configuration. It works slightly differently than this function. We could extract that into some shared utility module.